library/ssl_tls.c

```diff
@@ -826,7 +826,7 @@ static int ssl_populate_transform( mbedtls_ssl_transform *transform,
                                    const unsigned char randbytes[64],
                                    int minor_ver,
                                    unsigned endpoint,
-                                   const mbedtls_ssl_context *ssl )
+                                   mbedtls_ssl_context *ssl )
```

Comment:
As per the function description, remove const only when MBEDTLS_SSL_HW_RECORD_ACCEL is defined?